hieradata/workroom.yaml.example

```diff
@@ -1,9 +1,11 @@
 ### User configurable settings
 ## Place any user-configurable settings here.
 
-## StackStorm Versions
-st2::version: 0.10.0
-st2::mistral_git_branch: st2-0.10.0
+## StackStorm Versions. Only set these if you need
+## a specific version installed. Otherwise, you will get latest
+## stable
+# st2::version: 0.10.0
```

Comment:
Wrong versions in example? We don't have neither st2 0.10.0 or mistral branch st2-0.10.0.

BTW as someone who doesn't know how st2workroom works, I wonder what are the defaults and where are they? Sometimes expectations are defaults are commented in example configurations.